ssl/t1_trce.c

```diff
@@ -945,7 +947,7 @@ static int ssl_print_client_hello(BIO *bio, const SSL *ssl, int indent,
     BIO_printf(bio, "cipher_suites (len=%d)\n", (int)len);
     if (msglen < len || len & 1)
         return 0;
-    while (len > 0) {
+    while (len > 1) {
```

Comment:
On side note! (And side note means that no action is expected/required.) Original condition was not actually wrong, because len is verified to be even. With this in mind I'd say that `while (len >= 2)` would be more readable. One can even emphasize it with `/* len is always even */`.

Reply:
I did not see this check, so I guess I should remove this patch.
There are there loops with that kind of check.